dojo/tools/pip_audit/parser.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from dojo.models import Finding
+
+
+class PipAuditParser:
+
+    def get_scan_types(self):
+        return ["pip-audit Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "pip-audit Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import pip-audit JSON scan report."
+
+    def requires_file(self, scan_type):
+        return True
+
+    def get_findings(self, scan_file, test):
+
+        scan_data = scan_file.read()
+        try:
+            data = json.loads(str(scan_data, 'utf-8'))
+        except:
+            data = json.loads(scan_data)
+
+        findings = list()
+        for item in data:
+            vulnerabilities = item.get('vulns', [])
+            if vulnerabilities:
+                component_name = item['name']
+                component_version = item.get('version', None)
```

Comment:
```suggestion
                component_version = item.get('version')
```
`None` is the default value.

Reply:
done